app/services/metadata_cloud_service.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-require 'csv'
 
 class MetadataCloudService
```

Comment:
same deal for this guy, it looks to me like it has no state, if you make everything static, you won't need to instantiate